sdk/EVMC.js

```diff
@@ -278,14 +278,14 @@ class EVMC {
      * @param params array
      * @returns {Promise<*>}
      */
-    callContractMethod = async (methodName, params = []) => {
+    callContractMethod = async (methodName, params = [], callOpts = []) => {
```

Comment:
memo

Added argument to specify blocknumber

Reply:
ref
https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html#methods-mymethod-call